packages/peregrine/lib/talons/ProductFullDetail/useProductFullDetail.js

```diff
@@ -178,6 +181,12 @@ export const useProductFullDetail = props => {
     );
 
     const [{ cartId }] = useCartContext();
+    const [{ isSignedIn, currentUser }] = useUserContext();
+
+    const [formApi, setFormApi] = useState();
+
+    const wishlistId =
+        isSignedIn && currentUser.wishlist && currentUser.wishlist.id;
```

Comment:
Looking into this a bit, `wishlist` is deprecated, and you should instead use `wishlists` to get a list of available wishlist ids, and then we probably want to present the choice to the user in some form of UX.

Reply:
that mean 1 user have many wishlist ids? Then how can I know exactly id to add product to wishlist?

Reply:
> that mean 1 user have many wishlist ids?

Yep! You can have more than one, at least that's how the schema reads to me.

>  Then how can I know exactly id to add product to wishlist?

This is why we would need some UX direction. I imagine "add to wishlist" would be a button that, when clicked, would turn into a dropdown/select box.

Reply:
@sirugh will we have design for this dropdown/select box?

Reply:
Yes! I will add mocks to the linked issue asap.